src/LamePaulaMixer.cpp

```diff
@@ -190,8 +190,10 @@ void LamePaulaMixer::fillBuffer(void* buffer, udword bufferLen, PaulaPlayer *pla
                     }
                 }
             }  // for voices
+	    ms += 20;
         }
     }  // while bufferLen
+    return ms;
```

Comment:
Hmm. But with typical audio buffers this will never ever return anything other than 20 or 0. And neither value would tell how much audio data is in the filled buffer in terms of "ms". Calculating that externally would be much more accurate. Such as to show current playtime/songpos. An added method could help, though, and e.g. getSongPos() would return current playtime in ms.